_health-care/_js/_components/social-security-number.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import _ from 'lodash';
+
+import ErrorMessage from './error-message';
+
+class SocialSecurityNumber extends React.Component {
+  constructor() {
+    super();
+    this.state = {hasError: false};
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  componentWillMount() {
+    this.id = _.uniqueId();
+  }
+
+  handleChange() {
+    const ssn = this.refs.ssn.value;
+
+    if (!this.validate(ssn)) {
+      this.setState({hasError: true});
+    } else {
+      this.setState({hasError: false});
+    } 
+
+    this.props.onUserInput(ssn);
+  }
+
+  validate(field) {
+    return field !== '' && /^\d{3}-\d{2}-\d{4}$/.test(field);
+  }
+
+  render() {
+    const error_class = this.state.hasError ? "usa-input-error" : ""
+    let error_message = '';
+    if (this.state.hasError) {
+      error_message = <ErrorMessage message="Please put your number in this format xxx-xx-xxxx"/>
+    }
+    return (
+      <div>
+        <div className={`usa-input-grid usa-input-grid-medium ${error_class}`}>
+          <label className={`${error_class}-label`} htmlFor={this.id + "_ssn"}>Social Security Number
+            <span className="usa-additional_text">Required</span>
+          </label>
+          {error_message}
+          <input type="text" id={this.id + "_snn"} value={this.props.ssn} placeholder="xxx-xx-xxxx"
```

Comment:
It is when the `ErrorMessage` component is included... I'll have to figure out a way to include this conditionally.